cypress/e2e/web/createTechnology.js

```diff
@@ -117,17 +117,17 @@ describe('creating/editing technology', () => {
 			cy.findAllByText(/21,00/i).should('exist');
 			cy.findAllByText(/61,00/i).should('exist');
 
-			cy.get('[name="technologyCosts.costs.maintenence_costs_add_button"]').click();
-			cy.get('[name="technologyCosts.costs.maintenence_costs[0]_remove_button"').should(
+			cy.get('[name="technologyCosts.costs.maintenance_costs_add_button"]').click();
```

Comment:
Corrigi todos os lugares onde estava escrito `maintenence`

Reply:
The man is good